samples/subsys/usb/mass/Kconfig

```diff
@@ -45,6 +51,17 @@ config DISK_RAM_VOLUME_SIZE
 config MASS_STORAGE_DISK_NAME
 	default "NAND" if DISK_ACCESS_FLASH
 	default "RAM" if DISK_ACCESS_RAM
+	default "SD" if DISK_ACCESS_SDHC
```

Comment:
This only works if `CONFIG_FAT_FILESYSTEM_ELM=y`
https://github.com/zephyrproject-rtos/zephyr/blob/5396350701cfd70834c61bccafe8bf94cb018c68/subsys/disk/Kconfig#L134-L137

Reply:
This is intentional, this configuration provides access from device and host side.